db/models/commentsModel.js

```diff
@@ -10,3 +12,25 @@ exports.selectCommentsByArticleId = (articleId) => {
       return result.rows;
     });
 };
+
+exports.insertCommentByArticleId = (articleId, newComment) => {
+  const { username, body } = newComment;
+  if (!username || !body)
+    return Promise.reject({
+      status: 400,
+      msg: "Input should have username and body.",
+    });
+  return checkArticleExists(articleId)
+    .then(() => {
+      return checkUserExists(username);
```

Comment:
This 'checkUserExists' arguably should be handled with PSQL error handling, but this isn't mandatory